### DISCUSSION TITLE
Suggestion: Firefly III

### DISCUSSION THREAD

**Discussion by lounsberry** [tteck/Proxmox#556] [Ideas]
Firefly III is a self-hosted, open-source personal finance application. It's incredibly powerful and somewhat complicated installation, so it might be quite a project. 

Here's their main website: [https://demo.firefly-iii.org/](https://www.firefly-iii.org/
Here is their demo page: [https://demo.firefly-iii.org/](https://demo.firefly-iii.org/)
And lastly, here's the GitHub Repository: [firefly-iii/firefly-iii](https://github.com/firefly-iii/firefly-iii)

Thank you so much for your time! This has been an amazing experience and has greatly expanded my capabilities in self-hosting software. I have learned about so many different softwares just by browsing through the options that you have already created, so again I'm very thankful.

**Comment by towerhand** [comment]
Now that Mint is shutting down, a script for this program would be great.

https://docs.firefly-iii.org/how-to/firefly-iii/installation/self-managed/

**Comment by D3SL** [comment]
I just did this myself and it was definitely a bumpy ride. The official documentation didn't work for me and I was stuck until I found other people with some of the [same issues](https://gist.github.com/Engr-AllanG/34e77a08e1482284763fff429cdd92fa).

The two main brick walls I ran into were needing to install to `/var/www/html/` rather than `/var/www/`, and for some reason it just refused to work on Debian 12 but ran fine on Ubuntu 20.04.

**Comment by grzech1983** [comment]
Hello,

is there any chance for this script as a proxmox LXC?

Thanks

**Reply by towerhand** [reply]
I ended up trying and liking actualbudget.
